docs/appendices/release-notes/5.10.2.rst

```diff
@@ -78,3 +78,7 @@ Fixes
 - Fixed a performance regression introduced in :ref:`version_5.10.0` that
   caused ``CREATE SNAPSHOT`` to use more memory. That could cause the
   statement to fail with ``OutOfMemoryError`` under memory pressure.
+
+- Fixed an issue that prevented nodes from starting when upgrading from <= 5.7
```

Comment:
Actually - maybe my suggested change was wrong - can this also happen if going from 5.9 to 5.10 or 5.8 to 5.9, or does it only happen for tables created <= 5.7?

Reply:
How about `upgrading to >= 5.8`?

Reply:
sounds good

Reply:
Yes. 5.9 to 5.10 was how we first saw it. (will share slack link for reference.)

Reply:
Ah, all good. My browser had not updated with the newer comments.